sesman/chansrv/clipboard.c

```diff
@@ -312,7 +318,7 @@ get_atom_text(Atom atom)
     return g_last_atom_name;
 }
 
-/*****************************************************************************/
+/**************************login_retry***************************************************/
```

Comment:
Been testing the cut+paste a lot to test the change, guess how that happened... :) will fix.